modules/bridge/src/main/ts/ephox/bridge/components/menu/Container.ts

```diff
@@ -0,0 +1,37 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Result } from '@ephox/katamari';
+import { CardItem } from '../../api/Menu';
+import { autocompleteSchema } from './Autocomplete';
+import { CardItemsSpec } from './CardMenuItem';
+import { descriptionSchema } from './Description';
+import { imageSchema } from './Image';
+
+export interface ContainerSpec {
+  type: 'container';
```

Comment:
This name is going to clash if we ever want a more generic container (remember types aren't namespaced, they are global). I'd suggest renaming this to `cardcontainer` or something more specific.

Reply:
I renamed this to `cardcontainer`